src/Samples/SarifDeferredSample/Program.cs

```diff
@@ -69,20 +69,20 @@ private static void Main(string[] args)
                 int fileCount = 0;
                 int uriLengthTotal = 0;
 
-                if (run.Files != null)
+                if (run.Artifacts != null)
                 {
                     // Fastest: Enumerate
-                    foreach (FileData file in run.Files)
+                    foreach (Artifact artifact in run.Artifacts)
                     {
-                        uriLengthTotal += file?.FileLocation?.Uri?.OriginalString?.Length ?? 0;
+                        uriLengthTotal += artifact?.Location?.Uri?.OriginalString?.Length ?? 0;
```

Comment:
>Location [](start = 52, length = 8)

The reason it was `file.fileLocation` rather than just `file.location` was because `location` is that bigger object that contains `physicalLocation` and `logicalLocation`, and `physicalLocation` is `fileLocation` + `region`.